pkg/operator/ceph/cluster/mon/health.go

```diff
@@ -449,8 +449,8 @@ func (c *Cluster) determineExtraMonToRemove() string {
 	arbitraryMon := ""
 	for _, mon := range mons {
 		if mon.NodeName == "" {
-			logger.Debugf("mon %q is not scheduled to a specific host", mon.DaemonName)
-			continue
+			logger.Infof("mon %q is not scheduled to a specific host, removing portable mon %q", mon.DaemonName, mon.DaemonName)
+			return mon.DaemonName
```

Comment:
While your solution looks perfectly fine, I'll suggest we try setting the `arbitraryMon` var, in case other mons are actually assigned a node and the method might find a more preferred mon to remove instead of returning immediately.
```suggestion
			arbitraryMon = mon.DaemonName
			continue
```

Reply:
yeah that make sense compare to code below this.